app/_components/prose/_prose.scss

```diff
@@ -168,4 +169,21 @@
   .footnote-backref {
     text-decoration: none;
   }
+
+  .header-anchor {
+    text-decoration: none;
+    font-style: italic;
+    font-weight: normal;
+    color: govuk-colour("mid-grey");
+
+    &:hover {
+      color: $govuk-link-hover-colour;
+    }
+  }
+
+  .table-of-contents {
+    > ul > li {
```

Comment:
Can we use an explicit class for this?

Reply:
This is generated by the `markdown-it-table-of-contents` plugin; it provides the parent class, which keeps this scoped.